### ISSUE TITLE
Issue with Ngrook For Windows - 'The remote file either doesn't exist'

### ISSUE THREAD

**Issue by youngclaude** [ngrok/ngrok#51]
During NGrok installation via Chocolatey I got an error that says: 
```
Attempt to get headers for https://bin.equinox.io/a/c4Cafbnz7jm/ngrok-v3-3.14.1-windows-amd64.zip failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://bin.equinox.io/a/c4Cafbnz7jm/ngrok-v3-3.14.1-windows-amd64.zip'. Exception calling "GetResponse" with "0" argument(s): "The underlying connection was closed: An unexpected error occurred on a send."
Downloading ngrok 64 bit
  from 'https://bin.equinox.io/a/c4Cafbnz7jm/ngrok-v3-3.14.1-windows-amd64.zip'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://bin.equinox.io/a/c4Cafbnz7jm/ngrok-v3-3.14.1-windows-amd64.zip'. Exception calling "GetResponse" with "0" argument(s): "The underlying connection was closed: An unexpected error occurred on a send."
This package is likely not broken for licensed users - see https://docs.chocolatey.org/en-us/features/private-cdn.
```

Command Prompt Logs: 
![image](https://github.com/user-attachments/assets/e23f9f53-34bb-4c42-9069-66d09246d23b)

Attempt to directly access the resource in browser: 
![image](https://github.com/user-attachments/assets/0555b8f2-0138-42b8-819d-de514deda78a)

Note the downloadable files on https://ngrok.com/download are also down: 
![image](https://github.com/user-attachments/assets/62414015-9ed4-4439-a62a-b8474d3231f1)
![image](https://github.com/user-attachments/assets/eec43fa0-0604-4d58-a48a-5eaa0c6cc553)

Please advise. Thanks in advance.

**Comment by russorat** [comment]
@youngclaude thank you for the issue! we've been working with chocolately to try to get our binary unblocked but so far, they have been unresponsive. I've passed the SSL error to the team to investigate.

**Comment by russorat** [comment]
@youngclaude following up here on the SSL error, do you happen to have some sort of antivirus software running on your machine or network?

**Comment by youngclaude** [comment]
> @youngclaude following up here on the SSL error, do you happen to have some sort of antivirus software running on your machine or network?

No sir, not besides the default Windows Defender.

**Comment by russorat** [comment]
@youngclaude did you ever figure this out? we just released v3.18.1 so you can try with that one.

**Comment by russorat** [comment]
I'm going to close this out for now. If you are still having issues with the latest release, please open a new issue and we will take a look. Thanks!

**Comment by youngclaude** [comment]
@russorat Yes the resource is there now so the downloads are working. Thanks for you time on this.

**Comment by russorat** [comment]
great to hear!

**Comment by davequinn** [comment]
I am getting this issue now.

**Comment by russorat** [comment]
@davequinn thanks, but do you mind opening a new issue with what you're seeing?
